crates/lune-roblox/src/datatypes/conversion.rs

```diff
@@ -314,7 +315,8 @@ impl<'lua> LuaToDomValue<'lua> for LuaAnyUserData<'lua> {
                 value if value.is::<CFrame>()         => userdata_to_dom!(value as CFrame         => dom::CFrame),
                 value if value.is::<Color3>()         => userdata_to_dom!(value as Color3         => dom::Color3),
                 value if value.is::<ColorSequence>()  => userdata_to_dom!(value as ColorSequence  => dom::ColorSequence),
-                value if value.is::<Enum>()           => userdata_to_dom!(value as EnumItem       => dom::Enum),
+                value if value.is::<Enum>()           => userdata_to_dom!(value as EnumItem       => dom::EnumItem),
+                value if value.is::<EnumItem>()       => userdata_to_dom!(value as EnumItem       => dom::EnumItem),
```

Comment:
This looks.. weird, but I'm not 100% sure what would be correct. Was it even correct before?

Reply:
Yeah I stared at it for a long time because it didn't feel right. I don't think it was right before, because `rbx_types` doesn't have an equivalent to `Enum` (the roblox type).

I just changed it here because it was a necessary change to make Rust happy but it might need to just be removed?

Reply:
Yeah I think the `Enum -> EnumItem -> EnumItem` line should be removed. If I'm seeing things right, this will, when an `Enum` value is given to set some value in rbx-dom, try to cast it to a Lune-datatype `EnumItem` which will just always fail anyway.